generated-docs/React/Basic.md

```diff
@@ -372,7 +372,7 @@ caution.
 #### `toReactComponent`
 
 ``` purescript
-toReactComponent :: forall spec spec_ jsProps props state. Union spec spec_ (ComponentSpec props state) => ({  | jsProps } -> props) -> Component props -> { render :: Self props state -> JSX | spec } -> ReactComponent {  | jsProps }
+toReactComponent :: forall spec spec_ jsProps props state. Union spec spec_ (ComponentSpec props state) => (Record jsProps -> props) -> Component props -> { render :: Self props state -> JSX | spec } -> ReactComponent (Record jsProps)
```

Comment:
This seems odd, what version of the compiler are you using?

Reply:
```
sabaki$ purs --version
0.12.5
```

Reply:
Huh.. oh well :)